### ISSUE TITLE
After start charge session it takes 15mins before sensor.alfen_tag_socket_1 to update

### ISSUE THREAD

**Issue by barthoefs** [leeyuentuen/alfen_wallbox#123]
![image](https://github.com/leeyuentuen/alfen_wallbox/assets/89918293/d3f8c4bc-4d72-4db5-ba93-1a24e5fba8b3)

sensor.alfen_tag_socket_1 -> after 15 mins
sensor.alfen_transaction_socket_1_charging -> every 15/30mins
sensor.alfen_transaction_socket_1_charging_time -> every 20 mins

Is this by design?

![image](https://github.com/leeyuentuen/alfen_wallbox/assets/89918293/c791207a-0119-49b5-b777-5df303203ed9)

![image](https://github.com/leeyuentuen/alfen_wallbox/assets/89918293/79a433c9-9dc8-445b-a519-0dec6ab77c0e)

**Comment by leeyuentuen** [comment]
yes it is done on purpose. because the transaction of the wallbox give every 15 min transaction data. if i'm fetch them every 15sec. it would be heavy and it also create issue with overloading data. if i update every 10min. it only update the data on the 15th minutes

**Comment by GPoorten** [comment]
Hi,
I'm very happy with the addition of the tag info to the integration.
The time delay in update throws off my automation where i log the start kwh reading and the tag used to activate the charging session so i can create a usage report for declaration purposes.

Is it possible that this ID Tag value is updated when the state changes from "preparing" to "charging" or from "Available" "Preparing? 

Thanks a lot for the help.
Best regards,

Guido

**Comment by leeyuentuen** [comment]
> Hi, I'm very happy with the addition of the tag info to the integration. The time delay in update throws off my automation where i log the start kwh reading and the tag used to activate the charging session so i can create a usage report for declaration purposes.
> 
> Is it possible that this ID Tag value is updated when the state changes from "preparing" to "charging" or from "Available" "Preparing?
> 
> Thanks a lot for the help. Best regards,
> 
> Guido

i thought this is a limit of the alfen wallbox. the transaction will only update after 15min.

**Comment by GPoorten** [comment]
Hi,
Thanks for the quick reply. 
Hope this helps, the OCCP back-end integration does update when starting to charge.

![Charging tag update](https://github.com/user-attachments/assets/70c710ef-7765-40b5-8c19-4efbc6eb6416)

I'm trying to work arround that integration because i want to be able to switch to an official backend.

Best regards,

Guido
